components/create_comment/create_comment.test.tsx

```diff
@@ -435,7 +455,9 @@ describe('components/CreateComment', () => {
 
             // Note that setProps doesn't actually trigger componentDidUpdate
             wrapper.setProps(newProps);
-            wrapper.instance().componentDidUpdate(props, newProps);
+            // eslint-disable-next-line
+            // @ts-ignore
+            (wrapper.instance() as CreateCommentClass).componentDidUpdate(props, newProps);
```

Comment:
`componentDidUpdate` is expecting a state as second parameter. I saw similar usage (i,e, `componentDidUpdate(props, newProps);` ) in the codebase with `ts-ignore`. 

I couldnt find any way make this work so I followed the same approach.  Please let me know if you have any suggestion